docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

```diff
@@ -481,16 +475,26 @@ It is definitely too aggressive for person names but it is perfect for the book
 
 == Adding full text capabilities to our REST service
 
-In our existing `LibraryResource`, we just need to inject the following methods (and add a few ``import``s):
+In our existing `LibraryResource`, we just need to inject the `SearchSession`:
+
+[source,java]
+----
+    @Inject
+    SearchSession searchSession; // <1>
+----
+<1> Inject a Hibernate Search session, which relies on the `EntityManager` under the hood.
+Applications with multiple persistence units can use a CDI qualifier to select the right one:
+either `@Named` or `@io.quarkus.hibernate.orm.PersistenceUnit`.
```

Comment:
Not a blocker, but I'd prefer such a note to be fully expressed with an example.

Reply:
Ok, I'll add an example in the next PR that adds first-class support for multiple PUs.